helm/kyverno-policies-observability/tests/ats/test_common_default.py

```diff
@@ -60,3 +64,75 @@ def test_kubeadmconfig_auditpolicy(kubeadmconfig_with_audit_file) -> None:
     :param kubeadmconfig_with_audit_file: KubeadmConfig CR which includes an existing audit file
     """
     assert len(kubeadmconfig_with_audit_file['spec']['files']) == 1
+
+
+@pytest.mark.smoke
+def test_silence_heartbeat_policy(silence) -> None:
+    """
+    test_silence_heartbeat_policy tests defaulting of an empty Silence to check for the negative Heartbeat matcher.
+    :param silence: Any Silence CR.
+    """
+    matchers = silence['spec']['matchers']
+
+    for matcher in matchers:
+        if matcher['name'] == "alertname":
+            assert (matcher['value'] == "Heartbeat" and not matcher['isRegex'] and not matcher['isEqual'])
```

Comment:
I do not understand this, are we testing that no matcher are present??

Reply:
We are asserting that the silence CR gets added the matcher to let heartbeat through even if a whole installation is silenced